app/assets/stylesheets/core-layout.scss

```diff
@@ -1,4 +1,6 @@
 // import the base styles
 @import "header-footer-only";
 
+@import "helpers/mixins";
 @import "helpers/reset";
+@import "helpers/wrapper";
```

Comment:
Adding wrapper to the `core-layout` will make it harder to remove it – since it's currently required by only 2 applications, if we move it here it will potentially leak styles into any app.

Reply:
`#wrapper` was already included here because it was defined in `helpers/layouts`. What I've done is remove that file and take the only bit we need (`#wrapper`) and add it directly in here. Hopefully the cleaning up of the commits that I've just done will make this easier to read.